src/fetchers/tarball-fetcher.js

```diff
@@ -170,7 +172,17 @@ export default class TarballFetcher extends BaseFetcher {
           extractorStream,
         } = this.createExtractor(overwriteResolved, resolve, reject);
 
-        //
+        const handleRequestError = (res) => {
+          if (res.statusCode >= 400) {
+            // $FlowFixMe
+            const statusDescription = http.STATUS_CODES[res.statusCode];
```

Comment:
Good catch! I created an issue for it: https://github.com/facebook/flow/issues/2860